src/crackmes/hackthebox/find-the-easy-pass/fire-up-the-debugger.md

```diff
@@ -0,0 +1,83 @@
+# Fire up the debugger
+
+The next step will be to put a breakpoint at `0x0045412b`,
+this address contains probably the right password or our.
```

Comment:
Looks like this sentence got over in the middle?

Reply:
Yes, but the rendering seems correct  :
![rizin-page-served-01](https://user-images.githubusercontent.com/85988120/124504105-1118cb00-ddc7-11eb-825a-3f21922da6b9.png)

Reply:
Sorry, I should've been more clear on what I was pointing to.

What I was referring to is the line `this address contains probably the right password or our.`
Or our what? -- I think that sentence is sort of incomplete.